### PR TITLE
feat(TKC-2113): unTCL: TestWorkflow OpenAPI <-> CRD mappers

### PR DESCRIPTION
**Pull request by rangoo94**
## Pull request description 

* Move TestWorkflow Cloud connection instructions to MIT-licensed code
* Includes https://github.com/kubeshop/testkube/pull/5581
* Includes https://github.com/kubeshop/testkube/pull/5582
* Includes https://github.com/kubeshop/testkube/pull/5583

## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: cmd/testworkflow-init/data/global.go [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+import (
+	"os"
+
+	"github.com/kubeshop/testkube/pkg/expressions"
+	"github.com/kubeshop/testkube/pkg/expressions/libs"
+)
+
+func GetBaseTestWorkflowMachine() expressions.Machine {
+	var wd, _ = os.Getwd()
```
Comment (vsukhin) [inline cmd/testworkflow-init/data/global.go:11]: worth to log error

File: cmd/testworkflow-init/main.go [hunk 2]
```diff
@@ -104,13 +97,21 @@ func main() {
...
 		case constants.ArgDebug:
 			config["debug"] = os.Args[i+1]
 		default:
 			output.Failf(output.CodeInputError, "unknown parameter: %s", os.Args[i])
 		}
 	}
 
+	// Clean up unnecessary variables for non-toolkit containers
+	if !toolkit {
+		_ = os.Unsetenv("TK_REF")
```
Comment (vsukhin) [inline cmd/testworkflow-init/main.go:112]: worth to log error

File: cmd/testworkflow-toolkit/transfer/utils.go [hunk 3]
```diff
@@ -0,0 +1,12 @@
+package transfer
+
+import (
+	"crypto/sha256"
+	"encoding/json"
+	"fmt"
+)
+
+func SourceID(dirPath string, files []string) string {
+	v, _ := json.Marshal(map[string]interface{}{"p": dirPath, "v": files})
```
Comment (vsukhin) [inline cmd/testworkflow-toolkit/transfer/utils.go:10]: worth to log error

**Review by vsukhin — APPROVED** [review]

File: cmd/testworkflow-init/data/global.go (hunk 1 above)
Comment (rangoo94) [reply cmd/testworkflow-init/data/global.go:11]: I'll add it 👍

File: cmd/testworkflow-init/main.go (hunk 2 above)
Comment (rangoo94) [reply cmd/testworkflow-init/main.go:112]: I think that it should fail silently - it's not important for user

**Pull request merged**